tests/python/TestBoundingBox.py

```diff
@@ -0,0 +1,463 @@
+
+
+    def test_findBoundingBoxByCoordinates(self):
+        db = vdms.vdms()
+        db.connect(hostname, port)
+
+        all_queries = []
+
+        rect_coords = {}
+        rect_coords["x"] = 0
+        rect_coords["y"] = 0
+        rect_coords["width"] = 100
+        rect_coords["height"] = 100
+
+        results = {}
+        results["list"] = ["name"]
+
+        bb_params = {}
+        bb_params["rectangle"] = rect_coords
+        bb_params["results"] = results
+
+        query = {}
+        query["FindBoundingBox"] = bb_params
+
+        all_queries.append(query)
+
+        response, img_array = db.query(all_queries)
+
+        self.assertEqual(response[0]["FindBoundingBox"]["status"], 0)
+        # for i in range(0, len(response[0]["FindBoundingBox"]["entities"])):
+        #     print response[0]["FindBoundingBox"]["entities"][i]["name"]
+
+    def test_findBoundingBoxBlob(self):
+        db = vdms.vdms()
+        db.connect(hostname, port)
+
+        prefix_name = "my_brain_return_"
+        all_queries = []
+
+        for i in range(0, number_of_inserts):
+            db = vdms.vdms()
+            db.connect(hostname, port)
+
+            img_name = prefix_name + str(i)
+            imgprops = {}
+            imgprops["name"] = img_name
+            self.addBoundingBoxwithImage(db, 1, imgprops)
+
+        for i in range(0, number_of_inserts):
+            constraints = {}
+            constraints["name"] = ["==", prefix_name + str(i) + "_bb_0"]
+
+            results = {}
+            results["blob"] = True
+            results["list"] = ["name"]
+
+            img_params = {}
+            img_params["constraints"] = constraints
+            img_params["results"] = results
+
+            query = {}
+            query["FindBoundingBox"] = img_params
+
+            all_queries.append(query)
+
+        response, img_array = db.query(all_queries)
+
+        self.assertEqual(len(img_array), number_of_inserts)
+        for i in range(0, number_of_inserts):
+            self.assertEqual(response[i]["FindBoundingBox"]["status"], 0)
+
+    def test_updateBoundingBox(self):
+        db = vdms.vdms()
+        db.connect(hostname, port)
+
+        prefix_name = "update_bb_"
+
+        for i in range(0, number_of_inserts):
+            props = {}
+            props["name"] = prefix_name + str(i)
+            self.insertBoundingBox(db, props=props)
+
+        all_queries = []
+
+        constraints = {}
+        constraints["name"] = ["==", prefix_name + str(0)]
+
+        props = {}
+        props["name"] = "updated_bb_0"
+
+        bb_params = {}
+        bb_params["constraints"] = constraints
+        bb_params["properties"] = props
+
+        query = {}
+        query["UpdateBoundingBox"] = bb_params
+
+        all_queries.append(query)
+
+        response, img_array = db.query(all_queries)
+
+        self.assertEqual(response[0]["UpdateBoundingBox"]["count"], 1)
```

Comment:
here is checking that the call went well, but not that the actual update happened.